cogs/modmail.py

```diff
@@ -376,15 +375,25 @@ def obj(arg):
         ctx.message.content = msg
         thread = await self.bot.threads.find(channel=ctx.channel)
         if thread:
+            await ctx.trigger_typing()
```

Comment:
Why not `async with ctx.typing():`?

Reply:
There's no need for extra indentation, an asynchronous context manager would be suitable for long running operations.

Reply:
I just feel that it'll be typing after message sent

Reply:
That's not how discord typing works, typing stops after a message is sent or 10 seconds.